cfme/networks/cloud_network.py

```diff
@@ -25,6 +26,35 @@ def all(self):
         list_networks_obj = view.entities.get_all(surf_pages=True)
         return [self.instantiate(name=n.name) for n in list_networks_obj]
 
+    def create(self, name, auto_conf=True, tenant=None, provider=None, net_type=None, switch=None):
+        """Create method of cloud network
+           If autoconf is true, all options are automatically chosen if the given ones are not set
+            or do not exist (except net_type)
+           switch represents tuple of 3 booleans representing states of switches in the adding view,
+              switch = (external_router, administrative_state, shared)
+        """
+        view = navigate_to(self, 'AddNewCloudNetwork')
+        network_manager = "{}{}".format(provider.name, " Network Manager")
+        view.provider_dropdown.select_by_visible_text(network_manager)
+
+        view.name_field.fill(name)
+        if tenant:
+            view.tenant_select.select_by_visible_text(tenant)
+        elif auto_conf:
+            option = view.tenant_select.all_options
+            view.tenant_select.select_by_visible_text(option[1].text)
+
+        if net_type:
+            view.tenant_select.select_by_visible_text(net_type)
+
+        if switch:
+            view.external_router.fill(switch[0])
+            view.administrative_state.fill(switch[1])
+            view.shared.fill(switch[2])
```

Comment:
**REQURIED**
Here and below.
This looks odd, fill the view using method `fill`:
```python
view.fill({
"widget": "value"
...
})
```
`None` values will be skipped.